contrib/cabal.project

```diff
@@ -1,4 +1,5 @@
 packages:
+  ../fuzzyfind
```

Comment:
local copy of fuzzyfind as we haven't built the new packages following the [update](https://github.com/runarorama/fuzzyfind/pull/5) yet